src/Compilers/CSharp/Portable/Syntax/Syntax.xml

```diff
@@ -3515,6 +3514,7 @@
       <Kind Name="IdentifierToken"/>
     </Field>
     <Field Name="TypeParameterList" Type="TypeParameterListSyntax" Optional="true" Override="true"/>
+    <Field Name="ParameterList" Type="ParameterListSyntax" Optional="true" Override="true" />
```

Comment:
This doesn't look like a merge. This looks like a feature work and I would prefer that work to be done in a separate PR.